docs/reference/user/activation.md

```diff
@@ -0,0 +1,150 @@
+# Activation {#RefActivation}
+
+_Author: Artyom Kazak_
+
+---
+
+A user is called _activated_ they have a verified identity -- e.g. a phone number that has been verified via a text message, or an email address that has been verified by sending an activation code to it.
+
+A user that has been provisioned via single sign-on is always considered to be activated.
```

Comment:
Non-activated users are (mostly) a thing of the past: The API still supports them, so it's nice to still document them, these days our well-behaved clients never create inactive users - instead, activation codes are requested before a call to /register is made, bypassing the inactive state. 

Perhaps the docs could reflect this (put less emphasis on the non-activated users)? It's not `"most functionality is available to all users"` - non-activated users provide for a terrible user experience (they cannot initiate new conversations with anyone).

Reply:
> Non-activated users are (mostly) a thing of the past

Isn't every guestroom user a non-activated user?

Reply:
I have removed "most functionality is available to all users" and emphasised that non-activated users are only used for guestrooms.

Reply:
> Isn't every guestroom user a non-activated user?

Yes, that's true. There are two kinds of non-activated users:

* those that are also "ephemeral" (terminology used in the code) or  "guestroom users" (terminology used in some other contexts, e.g. blog posts), i.e those that call /register using only a name, but no email/phone. Those users cannot become activated later.
* those that are only non-activated without being ephemeral; i.e. those that call /register with a name and an email|phone. Those users can activate later after confirming their email|phone. But this possibility, while still supported by the API, is only used in old clients. (It could be considered "deprecated", though there might still be older versions of mobile clients that still use this flow)